meetups/10/notes.md

```diff
@@ -0,0 +1,31 @@
+#### Meetup Link:             
+
+https://www.facebook.com/events/689591491239526
+
+#### Raw Fb Live Feed:
+
+https://www.facebook.com/groups/rubydevelopersnepal/permalink/917824595042179/
+
+
```

Comment:
Add link to pics here:

Pictures from Sunday's #RubyMeetup event 
https://www.meetup.com/Nepal-Ruby-Users-Group/photos/all_photos/?photoAlbumId=28174196